composer.json

```diff
@@ -52,7 +52,8 @@
         "twig/twig": "~1.11",
         "zendframework/zend-stdlib": "~2.3",
         "zendframework/zendsearch": "@dev",
-        "doctrine/doctrine-fixtures-bundle": "2.2.*"
+        "doctrine/doctrine-fixtures-bundle": "2.2.*",
+        "phpdocumentor/reflection-docblock": "^2.0"
```

Comment:
it seems that appveyor (windows?) has problems with the version ^3.0 as we don't need it and 2.0 is also supported by the vendors this is ok for now.